README.md

```diff
@@ -25,6 +25,73 @@ The following features have been implemented:
+    --pid PID             PID of process to run perf on, if '-p/--perf' is not
+                            set, this value will be ignored and would not take any
+                            effection. Multiple PIDs can be set by using more than
+                            one --pid args. Default is None and means the whole
+                            system.
+    --tidb-proc           Collect perf data for PD/TiDB/TiKV processes instead
+                            of the whole system.
+    --perf-exec PERF_EXEC
+                            Custom path of perf executable file.
+    --perf-freq PERF_FREQ
+                            Event sampling frequency of perf-record, in Hz.
+    --perf-time PERF_TIME
+                            Time period of perf recording, in seconds.
+    -l, --log             Enable to include log files in output, PD/TiDB/TiKV
+                            logs are included by default.
+    --syslog              Enable to include system log in output, will be
+                            ignored if -l/--log is not set. This may significantly
+                            increase output size.
+    --config-file         Enable to include various config files in output,
+                            disabled by default.
+    --pd-host PD_HOST     The host of PD server. Default to localhost.
+    --pd-port PD_PORT     The port of PD API service, default to 2379.
+
+    Note that some options would decrease system performance.
+
+If calling `insight.py` with no argument provided, following information are collected and saved to `data` directory of current working directory:
+
+ - `collector`, providing system information, program versions, NTP status, partition statics and process statics of TiDB modules (if present), placed under `collector` sub-directory.
+ - PD API output of `config`, `diagnose`, `health`, `hotspot`, `labels`, `members`, `operators`, `regions`, `regionstats`, `schedulers` and `status`, placed under `pdctl` sub-directory in JSON format.
+
+Using `-o` can set another location to store output data other than the default `data`.
+
+### Examples
+
+A common combination of params that could be useful is: `insight.py -l --config-file`, with which the basic system information, TiDB modules' log files and several config files are collected.
+
+If you want to collect runtime information of a TiDB cluster from PD on a remote host (instead of running the scripts on that server), set `--pd-host` to the IP address or a resolvable hostname of that host. Additionally, if PD is listening on non-default port, use `--pd-port` to set it.
+
+System log is not collectd by default, even if `-l` is set. If needed, use `-l --syslog` to enable it. Both `syslog` and `systemd-journald` log are supported and automatically detected, note that there is no content filter available for log files, thus enabling log file collecting may leads to very large output size.
```

Comment:
-> The system log is not collected by default, even if `-l` is set. If needed, use `-l --syslog` to enable it. Both `syslog` and `systemd-journald` logs are supported and automatically detected. Note that there is no content filter available for log files, thus enabling log file collecting may lead to a very large output size.